kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -263,9 +285,9 @@ function locationHashChanged() {
   });
 }
 
-window.onhashchange = function () {
+$(window).one('hashchange', function( e ) {    
```

Comment:
> Yes, it has had no problems with switching in between so far. One thing I just found is that the Reporting menu does not show up in Discover until you select a Saved search though

Does it show up if you don't select a saved search but refresh the page?

Reply:
I have added a fix for that issue as well, now it does show up on base Discover page